pennylane/templates/embeddings.py

```diff
@@ -46,12 +73,115 @@ def AmplitudeEmbedding(features, wires, pad=None, normalize=False):
 
     Args:
         features (array): input array of shape ``(2^n,)``
-        wires (Sequence[int] or int): qubit indices that the template acts on
+        wires (Sequence[int] or int): :math:`n` qubit indices that the template acts on
         pad (float or complex): if not None, the input is padded with this constant to size :math:`2^n`
         normalize (Boolean): controls the activation of automatic normalization
 
     Raises:
         ValueError: if inputs do not have the correct format
+
+    .. UsageDetails::
+
+        Amplitude embedding encodes a normalized :math:`2^n`-dimensional feature vector into the state
+        of :math:`n` qubits:
+
+        .. code-block:: python
+
+            import pennylane as qml
+            from pennylane.templates import AmplitudeEmbedding
+
+            dev = qml.device('default.qubit', wires=2)
+
+            @qml.qnode(dev)
+            def circuit(f=None):
+                AmplitudeEmbedding(features=f, wires=range(2))
+                return qml.expval(qml.PauliZ(0))
+
+            circuit(f=[1/2, 1/2, 1/2, 1/2])
```

Comment:
Since this repeats the code example at the top of the docstring, I tend to lean towards removing the one at the top, and keeping this one

Reply:
Done